web/src/tippyjs.ts

```diff
@@ -740,17 +744,61 @@ export function initialize(): void {
         },
     });
 
+    function set_home_view_tooltip_content(instance: tippy.Instance): void {
+        const escape_navigates_to_home_view = user_settings.web_escape_navigates_to_home_view;
+        const home_view =
+            settings_config.web_home_view_values[user_settings.web_home_view].description;
+        instance.setContent(
+            ui_util.parse_html(render_org_logo_tooltip({home_view, escape_navigates_to_home_view})),
+        );
+    }
+
     tippy.delegate("body", {
         target: "#realm-navbar-wide-logo",
         placement: "right",
         appendTo: () => document.body,
+        onShow: set_home_view_tooltip_content,
+    });
+
+    tippy.delegate("body", {
+        target: "#message-list-navigation-home-view",
+        placement: "bottom",
+        trigger: "mouseenter",
+        delay: EXTRA_LONG_HOVER_DELAY,
+        appendTo: () => document.body,
+    });
```

Comment:
I think these can be declared in `message_list_navigation.ts`; generally it's nicer to have all the code for a feature in one place, and so I've been trying to avoid just having every tooltip go in here when it could just live in its own module.

Reply:
Sure, moved!